artists/views.py

```diff
@@ -2,44 +2,69 @@
 from django.shortcuts import render
 
 from albums.models import Album
-from albums.views import show
 
 from django.core.exceptions import ValidationError
 
+from django.views import View
+
 from .models import Artist
 
+from django.http import HttpResponse
+
+class Index(View):
```

Comment:
this view could be minimized into this:
```python
class Index(ListView):
    queryset = Artist.objects.prefetch_related('album_set')
    context_object_name = 'artists_list'
    template_name = 'artists/index.html'
```

Reply:
there are more operations than getting all artists , what prefetch do ?